stackstac/rio_reader.py

```diff
@@ -304,6 +304,7 @@ def __init__(
         dtype: np.dtype,
         fill_value: Union[int, float],
         rescale: bool,
+        scale_offset: Tuple[float, float],
```

Comment:
Can you update the `Reader` protocol to include this argument?

Reply:
You bet